modules/common-workflows/pages/workflow-sp-upgrade-webui.adoc

```diff
@@ -18,22 +18,22 @@ They are used for illustration purposes only.
 [role=procedure]
 
 
-. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for "SUSE Linux Enterprise Server 15 SP4 x86_64 (BETA)"
+. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for "SUSE Linux Enterprise Server 15 SP5 x86_64 (BETA)"
 . Select the recommended channels.
 +
 image::upgradesles01.png[scaledwidth=80%]
 . Click btn:[Add Products].
 +
 image::upgradesles02.png[scaledwidth=80%]
 . Navigate to menu:Systems[registered client > Software > SP Migration].
-  You will see two targets, SP3 and SP4. 
+  You will see two targets, SP4 and SP5. 
 +
 image::upgradesles03.png[scaledwidth=80%]
-. Select [literal]``SUSE Linux Enterprise Server 15 SP4 x86_64 (BETA)``.
+. Select [literal]``SUSE Linux Enterprise Server 15 SP5 x86_64 (BETA)``.
   This will expand further as shown below.
```

Comment:
Same as above, I would change anything in this recipe with the version change. Just state at the beginning that this recipe is from this version to this and users should take care of selecting correct channels in case they are migration from/to different version.

Reply:
I added this info to https://github.com/uyuni-project/uyuni-docs/issues/2294